platform/web/js/libs/library_godot_fetch.js

```diff
@@ -159,7 +167,10 @@ const GodotFetch = {
 		if (!obj.response) {
 			return 0;
 		}
-		if (obj.reader) {
+		// If the reader is nullish, but there is no body, and the request is not marked as done,
+		// the same status should be returned as though the request is currently being read
+		// so that the proper lifecycle closure can be handled in read()
+		if (obj.reader || (obj.response.body === null && !obj.done)) {
```

Comment:
Added a period, and changed the check from `===` to `==`.

```suggestion
		// If the reader is nullish, but there is no body, and the request is not marked as done,
		// the same status should be returned as though the request is currently being read
		// so that the proper lifecycle closure can be handled in `read()`.
		if (obj.reader || (obj.response.body == null && !obj.done)) {
```